qutebrowser/browser/commands.py

```diff
@@ -427,7 +427,7 @@ def tab_clone(self, bg=False, window=False):
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
                        maxsplit=0)
-    @cmdutils.argument('index', completion=miscmodels.other_buffer)
+    @cmdutils.argument('index', completion=miscmodels.OtherBuffer())
```

Comment:
The file you're referring to is slightly outdates. We now use Tabs instead of Buffer. Regardless, the concept is the same. What about something like TabFocus? Should that also be part of the same API? How would that look like?

Reply:
Maybe `Buffer(all_windows=false)`?
I'm also ok with them being separate classes too though.
I think the idea was to have very similar models be the same class with parameters to control the specifics